### ISSUE TITLE
release: Where are the MacOS ARM archives ...?

### ISSUE THREAD

**Issue by bep** [gohugoio/hugo#8952]
https://discourse.gohugo.io/t/hugo-0-88-released/34550/3?u=bep

**Comment by bep** [comment]
I see this in the log:

```bash
building binaries
         • DEPRECATED: skipped darwin/arm64 build on Go < 1.16 for compatibility, check https://goreleaser.com/deprecations/#builds-for-darwinarm64 for more info.
         • DEPRECATED: skipped darwin/arm64 build on Go < 1.16 for compatibility, check https://goreleaser.com/deprecations/#builds-for-darwinarm64 for more info.
```

Since I'm building with ... Go 1.17 this seems like a bug in the version logic in goreleaser, @caarlos0 ?

**Comment by caarlos0** [comment]
@bep yep, its fixed on latest versions

**Comment by caarlos0** [comment]
fwiw https://github.com/goreleaser/goreleaser/pull/2407

**Comment by bep** [comment]
I have "fixed" this in 0.88.1 by downgrading the build image, but keep this issue open to track the "proper fix".

**Comment by bep** [comment]
This was fixed in 625d2c257a8284515922f2235b9abf9ad28cc907 -- note that we have not released anything with that update yet, but I'm going to assume that it works (I have tested it)
